armi/resources/burn-chain.yaml

```diff
@@ -1,4 +1,7 @@
+# Spontaneous fission data is added from Table 5.1 of
+# "Application Guide to Neutron Multiplicity Counting" - LA-13422-M
```

Comment:
I appreciate the reference.

It would be good to have a central place for references in the repo; just a MD/RST file with references that we can point to from code.

Reply:
Thanks @john-science - References for nuclear data would be good. Sometimes having the references as close to the source is helpful. Maybe we should modify the `burn-chain.yaml` structure to support references?